content/influxdb3/core/query-data/influxql/basic-query.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Perform a basic InfluxQL query
+seotitle: Perform a basic InfluxQL query in InfluxDB Clustered
```

Comment:
```suggestion
seotitle: Perform a basic InfluxQL query in InfluxDB 3 Core
```
I assume we can't use `{{< product-name >}}` here?

Reply:
Actually, you totally can!